ORM/ORMFundamentals/Controllers/OrdersController.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using ORMFundamentals.Entities;
+using ORMFundamentals.Services;
+
+namespace ORMFundamentals.Controllers;
+
+[Route("api/[controller]")]
+[ApiController]
+public class OrdersController : ControllerBase
+{
+    private readonly IOrderService _orderService;
+
+    public OrdersController(IOrderService orderService)
+    {
+        _orderService = orderService;
+    }
+
+    [HttpGet]
+    public async Task<IEnumerable<Order>> GetAll()
+    {
+        return await _orderService.GetAll();
+    }
+
+    [HttpGet("{id}")]
+    public async Task<ActionResult<Order>> GetOrderById(int id)
+    {
+        var order = await _orderService.GetOrderById(id);
+
+        if (order == null)
+        {
+            return NotFound();
+        }
+
+        return order;
+    }
+
+    [HttpPost]
+    public async Task<ActionResult<Order>> Add(Order order)
+    {
+        await _orderService.Add(order);
+
+        return CreatedAtAction(nameof(GetOrderById), new { id = order.Id }, order);
+    }
+
+    [HttpPut("{id}")]
+    public async Task<IActionResult> Update(int id, Order order)
+    {
+        if (id != order.Id)
+        {
+            return BadRequest();
+        }
+
+        await _orderService.Update(order);
+
+        return NoContent();
```

Comment:
Just an advice: in RESTful APIs whenever you work with `PUT` HTTP method you should return entire updated entity. `NoContent` is acceptable only for `PATCH` or `DELETE` HTTP methods.